src/runtime_src/tools/xclbinutil/aie-pdi-transform/libinclude/cdo_common.h

```diff
@@ -0,0 +1,47 @@
+/******************************************************************************
+* Copyright (c) 2021 Xilinx, Inc.  All rights reserved.
+* SPDX-License-Identifier: MIT
+******************************************************************************/
+
+
+/*****************************************************************************/
+/**
+*
+* @file cdo_common.h
+*
+* This is the file which contains CDO common helpers.
+*
+* @note
+*
+******************************************************************************/
+#ifndef XCDO_COMMON_H
+#define XCDO_COMMON_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/***************************** Include Files *********************************/
+#include <stdio.h>
+/************************** Constant Definitions *****************************/
+/**************************** Type Definitions *******************************/
+/***************** Macros (Inline Functions) Definitions *********************/
+#define XCdo_Print(format, ...) printf("CDO: " format, ##__VA_ARGS__)
```

Comment:
Can we change this from printf to fprintf. That will give us an option to capture all the output in a log file. We can specify the log file name as third arg in  pdi_transform(). pdi_transform() should open the log file a setup the global file pointer used in XCdo_Print()?

Reply:
I can try to change printf to fprintf